### DISCUSSION TITLE
Keep .qmd files and not render them as part of a book

### DISCUSSION THREAD

**Discussion by mmmmtoasty19** [quarto-dev/quarto-cli#1803] [Q&A] [answered]
I am using Quarto to create my capstone project as a book.  I however would like to keep some additional .qmd files in the repo.  Is it possible to not render these when I render the book?  They seem to auto render when I click render in Rstudio.  Not a huge deal but just curious!

**Comment by jjallaire** [comment, marked as answer]
book projects should only render things listed in chapters (whereas website render everything by default)

You can control what is rendered in a website w/ an explicit render lists or by prefacing directories or files with underscore (`_`). See https://quarto.org/docs/websites/index.html#render-targets
